xbmc/platform/win32/filesystem/Win32File.cpp

```diff
@@ -761,3 +761,11 @@ int CWin32File::Stat(struct __stat64* statData)
 
   return 0;
 }
+
+int CWin32File::GetChunkSize()
+{
+  if (m_smbFile)
+    return 128 * 1024;
```

Comment:
In Windows `CWin32File` is common for SMB and regular local files calls as is used built-in OS SMB API.

`m_smbFile` flag differentiates if is SMB file or local file. Only returns 128K size for SMB and zero for local files as before.